org.eclipse.lsp4e/src/org/eclipse/lsp4e/LanguageServerWrapper.java

```diff
@@ -152,27 +151,33 @@ public void dirtyStateChanged(IFileBuffer buffer, boolean isDirty) {
 
 	};
 
+	private static class Context {
```

Comment:
Could `LanguageServerContext` be a better name?

If I am not mistaken, all the information in the class refers to the LanguageServer and the related communication channels.